api/src/main/java/com/velocitypowered/api/event/player/PlayerClientBrandEvent.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2018 Velocity Contributors
+ *
+ * The Velocity API is licensed under the terms of the MIT License. For more details,
+ * reference the LICENSE file in the api top-level directory.
+ */
+
+package com.velocitypowered.api.event.player;
+
+import com.google.common.base.Preconditions;
+import com.velocitypowered.api.event.ResultedEvent;
+import com.velocitypowered.api.event.connection.PluginMessageEvent;
+import com.velocitypowered.api.proxy.Player;
+
+/**
+ * Fired when a {@link Player} sends the <code>minecraft:brand</code> plugin message.
+ */
+public class PlayerClientBrandEvent implements ResultedEvent<PluginMessageEvent.ForwardResult> {
```

Comment:
I don't see any purpose to allow this event to be cancelled. Why would you want to do that?

Reply:
No clue! Honestly I just copied what most events did which was that they were cancelable. Can't think of a reason you'd want to cancel it :^)